currency_rate_update/views/res_currency_rate_provider.xml

```diff
@@ -44,6 +44,9 @@
                     <group name="options">
                         <group>
                             <field name="service"/>
+                            <div attrs="{'invisible':[('service','!=','ECB')]}" class="text-muted" colspan="2">
+                              European Central Bank exchange rate service is suited for companies using EUR as main currency.
```

Comment:
I presume that we don't need this note?

Reply:
Finally I think it'simportant for the user to know that currencies are converted base on EUR and that the currency exchange rate between foreign currencies might differ from real currencies exchange rates.

Reply:
Yet we're not doing double conversion, and since ECB provides 1 rate only, I assume it's bidirectional? Yet telling that may not be an issue in the end